### PR TITLE
Minor fixes for lst errors

### PR DESCRIPTION
**Pull request by myrlyn**
fixed conficts within deities and demigods, source loads cleanly with most other sources now.

### CODE REVIEW

**Comment by BahamutDragon** [comment]
Mate,

Was a bit surprised to see this message:  "Sorry, we could not display the entire diff because too many files (1,880) changed." 99% of what I could review appears to be file mode changes. "100644 → 100755". Why the file mode change?

The substitution class change fixed the issue of a non-existent ability (correct), then it just added a SAB tag (Something which would have prompted another class feature for the sub class)..What I am trying to figure out why the FlurryLVL variable was not increased by 10. That would properly display the Greater Flurry effect on the monk flurry of blows class feature

**Comment by myrlyn** [comment]
That would probably be a better way of doing it.  I'll fix it tonight.
I think the file mode change is the result of me working on a thumb drive
I'm moving between windows and Linux machines.  I can fix all the file
modes and resubmitting the pull request.

**Comment by myrlyn** [comment]
Various minor edits and PRECAMPAIGN tags to resolve unconstructed references for content that actually exists within the tree.

**Comment by myrlyn** [comment]
fixes for deities and demigods

**Comment by BahamutDragon** [comment]
Mate,

I appreciate the patches, though I think you will find that using branches is easier on seeing the files you actually worked on. Plus, it follows common git use courtesy practices.

I still have 1,880 files showing as altered or modified, no change there. Separate branches for each feature or bug is best to make it easy on the reviewer.

"Merge pull request #1 from BahamutDragon/master … " -- A little git usage 101, you do not need to do a pull request to pull from an upstream repo. 

I will merge these in. For the future please use branches.

File: data/35e/wizards_of_the_coast/campaign_settings/dragonlance/campaign/dcs_equipment.lst [hunk 1]
```diff
@@ -34,12 +34,16 @@ Staff of Magius		PROFICIENCY:WEAPON|Quarterstaff	TYPE:Weapon.Melee.TwoHanded.Sim
...
+Gerard's Armor		TYPE:Magic.Armor.Heavy.ArmorProfHeavy.ArmorType_Heavy.Suit.Specific		PROFICIENCY:ARMOR|Full Plate		COST:1500	WT:50		ACCHECK:-6												EQMOD:MWORKA.PLUS1A.STEEL	MAXDEX:1								SOURCELONG:Dragonlance Campaign Setting	SOURCESHORT:dcs	SOURCEPAGE:p.174	SPELLFAILURE:35		BONUS:COMBAT|AC|8|TYPE=Armor.REPLACE
+<<<<<<< HEAD
+Kang's Armor		TYPE:Magic.Armor.Light.ArmorProfLight.ArmorType_Light.Suit.Specific.Nonmetal	PROFICIENCY:ARMOR|Leather		COST:10	WT:15		ACCHECK:0												EQMOD:MWORKA.PLUS5A		MAXDEX:6								SOURCELONG:Dragonlance Campaign Setting	SOURCESHORT:dcs	SOURCEPAGE:p.179	SPELLFAILURE:10		BONUS:COMBAT|AC|2|TYPE=Armor.REPLACE	BONUS:EQMARMOR|SPELLFAILURE|-10|PRECLASS:1,Bard=1
+=======
+Kang's Armor		TYPE:Magic.Armor.Light.ArmorProfLight.ArmorType_Light.Suit.Specific.Nonmetal	PROFICIENCY:ARMOR|Splint Mail		COST:10	WT:15		ACCHECK:0												EQMOD:MWORKA.PLUS5A		MAXDEX:6								SOURCELONG:Dragonlance Campaign Setting	SOURCESHORT:dcs	SOURCEPAGE:p.179	SPELLFAILURE:10		BONUS:COMBAT|AC|2|TYPE=Armor.REPLACE	BONUS:EQMARMOR|SPELLFAILURE|-10|PRECLASS:1,Bard=1
+>>>>>>> efe2235fa822c96a14bbd2f7c7e2908631cac3c9
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/dragonlance/campaign/dcs_equipment.lst]: I trust you will fix the file conflict here?

File: data/35e/wizards_of_the_coast/campaign_settings/dragonlance/campaign/dcs_profs_weapon.lst [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+<<<<<<< HEAD
+Shadowstaff			TYPE:Exotic.Melee.ExoticMelee.TwoHanded.TwoHandedMelee.TwoHandedExoticMelee
+=======
 
-Shadowstaff		TYPE:Exotic
+# Misc for PCGen
+# Weapon Name                   Type
+
+Flail (Dire).COPY=Shadowstaff
+>>>>>>> efe2235fa822c96a14bbd2f7c7e2908631cac3c9
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/dragonlance/campaign/dcs_profs_weapon.lst]: And this file conflict. It is best to resolve conflicts before submitting, these will cause errors.

**Comment by myrlyn** [comment]
Sorry about that - still getting used to git.  Most of my previous
development work has been using svn.  I chmodded files back to the original
mode for the subsequent check-ins, I'm not sure why they're still showing
as modified.

**Pull request merged**